setup.cfg

```diff
@@ -46,6 +46,6 @@ install_requires =
 [options.extras_require]
 channels =
     Django>=2.2
-    channels==3.0.4
+    channels==2.4.0
```

Comment:
Looking at https://github.com/django/channels/blob/main/CHANGELOG.txt maybe it should stay `>=3.0, <3.1`?